### PR TITLE
[SIG-3288] Show an icon for the child incidents in the overview

### PR DESCRIPTION
**Pull request by aditudorache**
This PR marks with a grey `play` icon all the child incidents in the overview list

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js [hunk 1]
```diff
@@ -88,6 +88,32 @@ Td.propTypes = {
   children: PropTypes.node,
 };
 
+const ParentIconStyle = styled.span`
+  display: flex;
```
Comment (MauritsR) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js:92]: Not sure if these properties (display & direction) are necessary. As far as I can see, they don't provide a visual change.

**Review by MauritsR — CHANGES_REQUESTED** [review]

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js [hunk 2]
```diff
@@ -172,11 +198,8 @@ const List = ({
             return (
               <tr key={incident.id}>
                 <Td detailLink={detailLink}>
-                  {incident.has_children && (
-                    <Icon size={14} role="img" aria-label="Hoofdmelding">
-                      <FastForward />
-                    </Icon>
-                  )}
+                  {incident.has_children && <ParentIcon />}
+                  {(incident.has_parent && <ChildIcon />) || null}
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js:202]: good catch!

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js:92]: You are right! Thanks!

**Review by MauritsR — APPROVED** [review]
Looks good :)

**Pull request merged**